File: 백준/Bronze/28014. 첨탑 밀어서 부수기/첨탑 밀어서 부수기.java
import java.util.*;
import java.lang.*;
import java.io.*;

class Main {
	public static void main (String[] args) throws IOException {
	   BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
	   
	   String line1 = br.readLine();
	   int n = Integer.parseInt(line1);
	   
	   StringTokenizer st = new StringTokenizer(br.readLine());
	   
	   int prev = Integer.parseInt(st.nextToken());
	   int cnt = 1;
	   
	   for(int i = 1; i < n; i++){
            int cur = Integer.parseInt(st.nextToken());
            
            if(prev <= cur){
                cnt++;
            }
            
            prev = cur;
	   }
	   
	   System.out.println(cnt);
	}
}

File: 백준/Bronze/28014. 첨탑 밀어서 부수기/첨탑 밀어서 부수기.cc
#include <iostream>

using namespace std;

int main() {
    ios::sync_with_stdio(false);
    cin.tie(NULL);
    int n;
    cin >> n;
    
    int prev, cur;
    
    cin >> prev;
    int cnt = 1;
    
    for(int i = 1; i < n; i++){
        cin >> cur;
        
        if(prev <= cur){
            cnt++;
        }
        
        prev = cur;
    }
    
    cout << cnt;
    return 0;
}